accelerator/config.c

```diff
@@ -517,3 +520,34 @@ void ta_core_destroy(ta_core_t* const core) {
   logger_destroy_json_serializer();
   br_logger_release();
 }
+
+#define DOMAIN_SOCKET "/tmp/tangle-accelerator-domain-socket"
+#define START_NOTIFICATION "tangle-accelerator-started"
```

Comment:
Shall we make a list of available commands for IPC?

Reply:
I don't think we need longer strings. Instead, something like `TA-start` would make sense.